### ISSUE TITLE
Obscure way to encode 'g', 'm', and possibly 'v' without using the alphabet

### ISSUE THREAD

**Issue by azerum** [bluewings/dev-dad#9]
## TL;DR;

```js
(''[(![]+{})[+!![]+[+[]]]+([]+{})[+!![]]+([][[]]+[])[+!![]]+(![]+[])[!+[]+!![]+!![]]+(!![]+[])[+[]]+(!![]+[])[+!![]]+([][[]]+[])[+[]]+(![]+{})[+!![]+[+[]]]+(!![]+[])[+[]]+([]+{})[+!![]]+(!![]+[])[+!![]]]+'')[+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]] 
```
```
> "g"
```

```js
((+[])[(![]+{})[+!![]+[+[]]]+([]+{})[+!![]]+([][[]]+[])[+!![]]+(![]+[])[!+[]+!![]+!![]]+(!![]+[])[+[]]+(!![]+[])[+!![]]+([][[]]+[])[+[]]+(![]+{})[+!![]+[+[]]]+(!![]+[])[+[]]+([]+{})[+!![]]+(!![]+[])[+!![]]]+'')[+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]] 
```
```
> "m"
```

## Explanation

This is related to your amazing [Writing a sentence without using the Alphabet](https://bluewings.github.io/en/writing-a-sentence-without-using-the-alphabet/) post

To get letter 'g', we can print the source code of `String` constructor

```js
''.constructor + ''

> "function String() {
    [native code]
}"
```

and then get the character under index 14. The full unobscure code will look like this:

```
(''.constructor + '')[14]

> "g"
```

Obscure version explanation:

```
// Word "constructor" encoded using your site
const theConstructor = (![]+{})[+!![]+[+[]]]+([]+{})[+!![]]+([][[]]+[])[+!![]]+(![]+[])[!+[]+!![]+!![]]+(!![]+[])[+[]]+(!![]+[])[+!![]]+([][[]]+[])[+[]]+(![]+{})[+!![]+[+[]]]+(!![]+[])[+[]]+([]+{})[+!![]]+(!![]+[])[+!![]];

// 1 + 1 + 1 + 1 .... to get 14
const theFourteen = +!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![];

(''[theConstructor]+'')[theFourteen]

> "g"
```

'm' can be obtained in similar fashion, using the source code of `Number`, where `m` is a character with index 11 

## P.S.

We could also get 'v' by taking characters from '[native code]', but I am not sure if all browsers are required by standard to print that string
